src/objs/basic_types.md

```diff
@@ -0,0 +1,53 @@
+## Basic Object Types
+In Object Oriented programming, objects inherit behaviour from parents. DM has some built in ancestors which share certain behaviours. These will be gone over individually and in more detail in their own pages, but here is the general overview.
+### Datums
+The first is the Datum (`/datum/`). It's the ancestor of all the other types (except for /world, /client, /list, and /savefile) that we will see. It's essentially pure data, hence the name, and can have pretty much any vars you like. When you define a new "top level" object, if you do not specify a parent_type, it defaults to /datum.
+```dm
+MyType
+	var/myvar = "test"
+// this "mytype object" is a datum. Normally, though, you'd put it as datum/MyType or /datum/MyType
+```
+### Atoms
+Atoms (`/atom/`) are the direct child of datums, and they represent objects that can go on a map and therefore in the world. It's an abstract parent which shouldn't really be used by itself, but it gives important information to its children, mainly areas, turfs, objs, and mobs.
+`/atom/movable` defines behaviour related to moving things, and is the parent of `/obj` and `/mob`. Again, this is an abstract parent, so you shouldn't define objects of its type.
+#### Areas
+An area (`/area/`) is how the game controls rooms and certain zones. Every tile on a map has exactly one area, and if you don't set one in the map editor, it defaults to `/area/space`.
+When the game starts, one object is created per area, and it's treated as a single object. All tiles with the same area are connected to the same single instance of that area.
```

Comment:
> When the game starts, one object is created per area

wat? are you trying to say `/area/foo` is a singleton? this is unclear wording. Don't toss 'object' around because it's confusing

Reply:
ah meant to say instance. was looking at the dm docs for reference and i didn't fully get it
![image](https://github.com/spacestation13/DMByExample/assets/51260013/f211be5a-b081-4872-871a-0d050cd1a14c)

Reply:
ended up just removing that bit altogether. That can go in area's own page later if need be.